packages/driver/cypress/e2e/e2e/origin/commands/misc.cy.ts

```diff
@@ -37,10 +37,22 @@ context('cy.origin misc', { browser: '!webkit' }, () => {
   })
 
   it('.pause()', () => {
+    // ensures the 'paused' event makes it to the event-manager in the primary.
+    // if we get cross-origin cy-in-cy test working, we could potentially make
+    // this even more end-to-end: test out the reporter UI and click the
+    // resume buttons instead of sending the resume:all event
+    Cypress.primaryOriginCommunicator.once('paused', ({ nextCommandName, origin }) => {
```

Comment:
Does the `consoleProps.pause` test in this file also need to be updated?

Reply:
I think it's okay to leave it as is. It basically short-circuits the communication with the runner, which is fine for the sake of testing out the consoleProps.

Reply:
I don't think so since the test only handles the interactive case and I think the only think that actually gets put out in the consoleProps is the command name when running in open mode.